parser-typechecker/src/Unison/Typechecker/Context.hs

```diff
@@ -2952,6 +2953,15 @@ isSubtype' type1 type2 = succeeds $ do
   appendContext (Var <$> vars)
   subtype type1 type2
 
+fitsScheme :: (Var v, Ord loc) => Type v loc -> Type v loc -> Either (CompilerBug v loc) Bool
```

Comment:
Docs please 😄 , some of this typechecker stuff gets tricky so I care a bit more about docs for these.